source/GameInterface/Services/MobileParties/Patches/MobilePartyPropertyPatches.cs

```diff
@@ -321,17 +302,16 @@ private static bool SetIsDisbandingPrefix(MobileParty __instance, bool value)
                 + "Callstack: {callstack}", nameof(MobileParty.IsDisbanding), Environment.StackTrace);
             return false;
         }
-
         var message = new MobilePartyPropertyChanged(PropertyType.IsDisbanding, __instance.StringId, value.ToString());
         MessageBroker.Instance.Publish(__instance, message);
-
         return ModInformation.IsServer;
     }
 
     [HarmonyPatch(nameof(MobileParty.CurrentSettlement), MethodType.Setter)]
     [HarmonyPrefix]
-    private static bool SetCurentSettlementPrefix(MobileParty __instance, Settlement value)
+    private static bool SetCurrentSettlementPrefix(MobileParty __instance, Settlement value)
     {
+        if (value == __instance._currentSettlement) return false;
```

Comment:
Please revert this file, the new lines make it easier to read
But please keep the new lines and [this update](https://github.com/Bannerlord-Coop-Team/BannerlordCoop/pull/998/files#diff-591d596d28072d88f4e09d9a435374d77eebb1e11f5a9ea3f97279bb7b24f063R101) as well